### PR TITLE
Initial setup RP job

### PR DESCRIPTION
**Pull request by ScottDormand96**
https://eaflood.atlassian.net/browse/IWTF-3381

Setup recurring payment job

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ packages/
   > Handles processing of post-office counter sales
 - [fulfilment-job](packages/fulfilment-job/README.md)
   > Handles processing of licence fulfilment requirements
+- [recurring-payments-job](packages/recurring-payments-job/README.md)
```
Comment (irisfaraway) [inline README.md:73]: Can we also have a line of description of what the job does, like the others do above?

**Review by irisfaraway — CHANGES_REQUESTED** [review]
Looks good, just a few little things to maybe tweak.

Also just wondering as the AC says we should be able to successfully run the job – do we need to have it outputting a line to the console or something just so we have something to test?

File: packages/recurring-payments-job/LICENSE [hunk 2]
```diff
@@ -0,0 +1,8 @@
+The Open Government Licence (OGL) Version 3
+
+Copyright (c) 2020 DEFRA
```
[the remark this answers is not in the document]
Comment (jaucourt) [reply packages/recurring-payments-job/LICENSE:3]: Let's check with a Principal Dev on this

[the remark this answers is not in the document]
Comment (ScottDormand96) [reply packages/recurring-payments-job/LICENSE:3]: I will message Ben

**Pull request merged**